bmwqemu.pm

```diff
@@ -158,6 +158,9 @@ sub ensure_valid_vars () {
 
     die "CASEDIR variable not set, unknown test case directory" if !defined $vars{CASEDIR};
     die "No scripts in CASEDIR '$vars{CASEDIR}'\n" unless -e $vars{CASEDIR};
+    if (defined $vars{WHEELS_DIR} && !-d $vars{WHEELS_DIR}) {
+        die "WHEELS_DIR '$vars{WHEELS_DIR}' does not exist";
+    }
```

Comment:
I'd suggest to use post-condition to be consistent with the above two lines.

```suggestion
    die "WHEELS_DIR '$vars{WHEELS_DIR}' does not exist" if defined $vars{WHEELS_DIR} && !-d $vars{WHEELS_DIR};
```

Reply:
done